app/posts/publish-teacher-training-courses/2018-08-14-formatting.md

```diff
@@ -23,7 +23,7 @@ The problem with these sorts of links is that they aren’t accessible. They don
 
 ## HTML
 
-We could allow users to use raw HTML: <a href="/blah">Link</a>. This is prone to more user error than markdown.
+We could allow users to use raw HTML: <a href="#">Link</a>. This is prone to more user error than markdown.
```

Comment:
Is this getting rendered as HTML, probably should encode the `<>` etc instead.

Reply:
Using inline code to format this example now.